docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -213,6 +218,63 @@ inside the container, so you can simply run:
 
 5. Stop and remove container
 
+.. code-block:: bash
+
+    docker stop pelux-build
+    docker rm pelux-build
+
+Particular layer build procedure:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Clone the pelux-manifests git repository with submodule
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/Pelagicore/pelux-manifests.git
+    cd pelux-manifests
+
+2. Copy layer directory to pelux-manifests
+
+.. code-block:: bash
+
+    cp -r ../<LAYER_NAME> .
+
+3. Build and run Docker container
+
+.. note:: If image was build previously you can reuse it and omit build command
+          To verify image existence please run 'docker images pelux'
+
+.. code-block:: bash
+
+    docker build --build-arg userid=$(id -u) --build-arg groupid=$(id -g) -t pelux .
+    docker run -t -d --name pelux-build -v $(pwd):/docker -w /docker pelux /bin/bash
+
+4. Run inside the Docker container
+
+At this point, we recommend using ``docker exec pelux-build <command>`` and to follow the same
+instructions as when building locally (but inside the Docker container).
+
+.. note:: You need ~40GB of free space in working directory
+
+.. code-block:: bash
+
+    docker exec pelux-build bash -c "/docker/build-in-docker.sh -v <VARIANT> -i <IMAGE> -b <BRANCH> -l <LAYER_NAME>"
+
+IMAGE and VARIANT options mentioned above
+LAYER_NAME is name of directory we copied in step 2
+BRANCH is optional; default - master
+
+5. Move the built images to the host
```

Comment:
If the script build-in-docker.sh is updated by adding line
```
mkdir -p /docker/output
cd /docker/output
```
before the repo init like this:
```
mkdir -p /docker/output
cd /docker/output
repo init -u https://github.com/Pelagicore/pelux-manifests.git -b $BRANCH;
repo sync;
```
then there is no need to have this step, or could be changed to create a symlink to output/build/tmp/deploy/images.

The main advantage of this is that you are able to debug much easier since you have direct access to logs and other artifacts in build/tmp/. You don't need to be in the container.

Consider to update the script with this change and then either remove or update this step since a copy is not needed.

Reply:
Not to mention the fact that overlay2fs is buggy. So much better to put build directory directly on file system on bind mount (e.g. the problems we had in CI and saw problems with overlay2fs again recently on a local setup).

Reply:
Build dir is already on host system. Just forgot to create a symlink instead of copying